src/HypothesisTests.jl

```diff
@@ -101,6 +108,9 @@ end
 # parameter of interest: name, value under h0, point estimate
 population_param_of_interest{T<:HypothesisTest}(test::T) = ("not implemented yet", NaN, NaN)
 
+# is the test one- or two-sided
+tail_of_distribution{T<:HypothesisTest}(test::T) = :undefined
```

Comment:
It doesn't really matter, but just as a general heads up, in cases like this the typevar isn't necessary; this could just be `tail_of_distribution(test::HypothesisTest)`. Not necessary to change it, just an FYI.